### ISSUE TITLE
Likes: Add context for assistive devices?

### ISSUE THREAD

**Issue by LeaVerou** [LeaVerou/bluesky-likes#9]
Spotted by @DmitrySharabin. Currently the Likes component is announced as e.g. (assuming 15 likes) "Butterfly fifteen" which is not super clear. OTOH, the way these components are meant to be used, the context should be obvious from usage (e.g. see the example). So perhaps all it needs is a note explaining this?

Otherwise, if we add our own `aria-label` on the `<a>` or different `alt` on the icon, we risk to have subpar a11y for the cases where the context *is* obvious. We can expose attributes for them, but that increases complexity sufficiently (e.g. now we need to expose localization for them) and folks can always slot a different icon in.

Thoughts? 🤔
